nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/test/resources/OSGI-INF/notification-event-listener-contrib.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.event.listener.contribution.test">
+
+  <extension target="org.nuxeo.ecm.platform.ec.notification.service.NotificationService"
+             point="generalSettings">
+    <settings>
+      <serverPrefix>http://localhost:8080/nuxeo/</serverPrefix>
+      <eMailSubjectPrefix>[Dummy]</eMailSubjectPrefix>
+      <mailSessionJndiName>java:comp/env/Mail</mailSessionJndiName>
+    </settings>
+  </extension>
+
+  <!-- define the listener that will be responsible for sending the mails -->
```

Comment:
Not clear, you're contributing a notification, not a listener.
I would remove the comment, it's self explanatory.